notes/2017/2017-08-31.md

```diff
@@ -0,0 +1,69 @@
+# 31-Aug-2017 ESLint TSC Meeting Notes
+
+## Transcript
+
+https://gitter.im/eslint/tsc-meetings/archives/2017/08/31
+
+## Attending
+
+* Teddy Katz (@not-an-aardvark) - TSC
+* Brandon Mills (@btmills) - TSC
+* Ilya Volodin (@ilyavolodin) - TSC
+* Kai Cataldo (@kaicataldo) - TSC
+* Gyandeep Singh (@gyandeeps) - TSC
+
+## Topics
+
+### [no-console rule should ignore assignment](https://github.com/eslint/eslint/issues/7806)
+
+* TSC Summary: This issue proposes making `no-console` less strict by only having it report *calls* to `console` methods, rather than reporting any property access on `console`.
+* TSC Question: Should we accept this issue?
+* @nzakas' [comment](https://github.com/eslint/eslint/issues/7806#issuecomment-269518852) suggested an option for this, enabled by default.
+* This could also be accomplished using `no-restricted-syntax`, so we could just add a note to the docs.
+* Everyone in attendence was ambivalent, so adding a note about `no-restricted-syntax` to the docs involves the least code.
+* :+1: to noting the `no-restricted-syntax` alternative in the docs from @ilyavolodin, @not-an-aardvark, @btmills, @kaicataldo, @gyandeeps
+
+**Resolution**: We will not accept the code enhancement, preferring instead to note the `no-restricted-syntax` alternative in the documentation.
+
+### [Using ESlint in the browser](https://github.com/eslint/eslint/issues/8348)
+
+* TSC Summary: This issue proposes adding official support for running ESLint in a browser. It seems like only a small change would be needed for this to work.
+* TSC Question: Should we accept this issue?
+* Browserify already works, but changes need to be made to work with Webpack.
```

Comment:
To clarify my comment in the meeting -- I think browserify doesn't work yet, since it would require [this change](https://github.com/eslint/eslint/issues/8348#issuecomment-302843526) to be made. I meant that additional changes beyond that one would be required to support Webpack.

Reply:
Thank you for clarifying! I'll update that line shortly.